microsoft-edge/progressive-web-apps-chromium/how-to/handle-urls.md

```diff
@@ -18,7 +18,7 @@ Progressive Web Apps (PWAs) can also handle links in a similar way.
 <!-- ====================================================================== -->
 ## Automatic link handling
 
-When you install a PWA using Microsoft Edge, all the links that refer to content within the scope of this PWA will automatically launch it instead of the web browser.
+When Microsoft Edge is your default browser and you install a PWA using Microsoft Edge, all the links that refer to content that's within the scope of this PWA will automatically launch it instead of the web browser.
```

Comment:
```suggestion
When Microsoft Edge is your default browser and you install a PWA using Microsoft Edge, all the links that refer to content that's within the scope of this PWA will automatically launch the PWA instead of launching the web browser.
```
i try to avoid the indirection pointer "it"